### PR TITLE
doc: what are the default values

### PR DESCRIPTION
**Pull request by tshepang**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -76,19 +76,25 @@ impl SessionMiddleware {
     }
 
     /// Set a value for the iss (issuer) jwt claim.
+    ///
+    /// The default is None.
```
Comment (kaj) [inline src/lib.rs]: "The default is to not set an issuer", would perhaps be more appropriate, as the argument isn't an Option and therefore cannot be None.  The default for the internal variable is indeed None, but that is internal.

**Review by kaj — CHANGES_REQUESTED** [review]
Thanks, stating the default values is indeed a good thing to do in the docstrings.

I have a comment on the first one, the two others are fine.

**Comment by tshepang** [comment]
review addressed

**Comment by kaj** [comment]
Merged.  Thanks!

**Pull request merged**